configure -byte-only is broken

<!-- Thank you for reporting a bug to Coq! -->

#### Description of the problem

The `-byte-only` option of the configure script is broken.
```
$ ./configure -profile=devel -byte-only
$ make
...
DUNE      _build/install/default/lib/stublibs/dllbyterun_stubs.so
Error: Don't know how to build
_build/install/default/lib/stublibs/dllbyterun_stubs.so
make[1]: *** [Makefile.common:165: _build/install/default/lib/stublibs/dllbyterun_stubs.so] Error 1

Error: Don't know how to build
_build/install/default/lib/stublibs/dllbyterun_stubs.so
```

#### Coq Version

branch v8.14 and master

--- a/Makefile.build
+++ b/Makefile.build
@@ -233,7 +233,7 @@ BOOTCOQC=$(TIMER) $(COQC) -coqlib $(VO_OUT_DIR) -q $(COQOPTS) $(DUNEPLUGININCLUD
 VO_TOOLS_DEP = $(COQC)
 
 ifeq ($(BEST),byte)
-  VO_TOOLS_DEP += $(CONTEXT)/lib/stublibs/dllbyterun_stubs.so
+  VO_TOOLS_DEP += $(CONTEXT)/lib/stublibs/dllcoqrun_stubs.so
 endif
 
 ifdef NATIVECOMPUTE

